packer/install-dependencies.sh

```diff
@@ -3,19 +3,19 @@ source /home/packer/provision_installs.sh
 source /home/packer/provision_source.sh
 source /home/packer/packer_source.sh
 
-RELEASE_NOTES_FILEPATH=/var/log/azure/golden-image-install.complete
+VHD_LOGS_FILEPATH=/opt/azure/vhd-install.complete
```

Comment:
Just to be clear, the rename of this var is just to better describe it?

Reply:
correct, and I used the same var in CSE so it's clear that they are the same thing and should both be updated if one of them changes.